### PR TITLE
Feat: moderation UI updates

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- updates to the moderation app UI

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/moderation/src/components/content-card/content.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import moment from 'moment';
```
Comment (SeverS) [inline ui/apps/moderation/src/components/content-card/content.tsx:2]: Do we really need moment? I think we have day.js

File: ui/apps/moderation/src/components/transparency-log/banner.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import DS from '@akashaorg/design-system';
-import { ICount } from '@akashaorg/ui-awf-hooks';
+import { ICount } from '@akashaorg/typings/ui';
```
Comment (SeverS) [inline ui/apps/moderation/src/components/transparency-log/banner.tsx]: count of what?

File: ui/apps/moderation/src/components/transparency-log/detail-card.tsx [hunk 3]
```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import DS from '@akashaorg/design-system';
-import { getMediaUrl, ILogItem } from '@akashaorg/ui-awf-hooks';
-import { ModerationItemTypes, NavigateToParams } from '@akashaorg/typings/ui';
+import { getMediaUrl } from '@akashaorg/ui-awf-hooks';
+import { ILogItem, ModerationItemTypes, NavigateToParams } from '@akashaorg/typings/ui';
```
Comment (SeverS) [inline ui/apps/moderation/src/components/transparency-log/detail-card.tsx]: Do you mean IModerationLogItem ?

File: ui/apps/moderation/src/components/content-card/content.tsx (hunk 1 above)
Comment (josenriagu) [reply ui/apps/moderation/src/components/content-card/content.tsx:2]: A couple of components outside design system use moment. I can clean that up to use dayjs and remove moment entirely, in a separate PR

File: ui/apps/moderation/src/components/transparency-log/banner.tsx (hunk 2 above)
Comment (josenriagu) [reply ui/apps/moderation/src/components/transparency-log/banner.tsx]: Moderation log items. I'll update the name now

File: ui/apps/moderation/src/components/transparency-log/detail-card.tsx (hunk 3 above)
Comment (josenriagu) [reply ui/apps/moderation/src/components/transparency-log/detail-card.tsx]: Yes exactly. These typings were defined and used within the moderation app initially before I moved them into the typings package. I'll update their names now to make them more self-explanatory

**Review by kenshyx — APPROVED** [review]

**Review by SeverS — APPROVED** [review]

**Pull request merged**